music_assistant/providers/opensubsonic/sonic_provider.py

```diff
@@ -827,7 +829,7 @@ def _streamer() -> None:
                     timeOffset=seek_position,
                     estimateContentLength=True,
                 ) as stream:
-                    for chunk in stream.iter_content(chunk_size=40960):
+                    for chunk in stream.iter_content(chunk_size=4096):
```

Comment:
Any particular reason to choose such a small chunk size ?
I would rather raise it then lower it. Something like 32kb or even higher.

Reply:
Damn, that was not supposed to be left in (I was testing to see if different chunk sizes changed seek performance, they don't AFAICT). We currently use 40Kb as a chunk size and I'll leave that alone if you agree.